src/inject/utils/log.ts

```diff
@@ -1,10 +1,10 @@
-declare const __DEBUG__: boolean;
-const DEBUG = __DEBUG__;
+declare const __WATCH__: boolean;
```

Comment:
DEBUG should be still there. DEBUG is used for logging info while you debug, WATCH is used for watching for code changes.

Reply:
.... This getitng me headache